sql/parser/sql.y

```diff
@@ -1003,13 +1005,19 @@ set_reset_clause:
   SET set_rest {}
 
 show_stmt:
-  SHOW var_name
+  SHOW var_name on_privilege_target_clause for_grantee_clause
```

Comment:
although my hack fails in the same way as "SHOW FOO", meaning we get an empty statement rather than a syntax error.